edm4hep.yaml

```diff
@@ -503,6 +503,22 @@ datatypes:
       void setCovMatrix(float value, edm4hep::Cartesian dimI, edm4hep::Cartesian dimJ) { getCovMatrix().setValue(value, dimI, dimJ); }\n
       "
 
+  edm4hep::DriftChamberDigi:
+    Description: "Digitized drift chamber hit (before tracking)"
+    Author: "EDM4hep authors"
+    Members:
+      - uint64_t cellID // ID of the sensor that created this hit
+      - int32_t type // type of the raw data hit
+      - int32_t quality // quality bit flag of the hit
+      - float time [ns] // time of the hit
+      - float eDep [GeV] // energy deposited on the hit
+      - float eDepError [GeV] // error measured on eDep
+      - edm4hep::Vector3d position [mm] // point on the sensitive wire (SW) which is closest to the simhit
+      - edm4hep::Vector3d directionSW // direction of SW
+      - float distanceToWire [mm] // distance hit-wire
+      - uint32_t nCluster // number of clusters associated to this hit
```

Comment:
Is this really needed if the vector member exists anyway?
Add a helper function to return nCluster from `nElectrons.size()`?

Reply:
Done